### PR TITLE
various minor cleanups

### PR DESCRIPTION
**Pull request by fw-immunant**
Shouldn't be contentious, will merge if CI passes.

### CODE REVIEW

File: libia2/include/ia2.h [hunk 1]
```diff
@@ -205,33 +209,38 @@ asm(".macro mov_mixed_pkru_eax pkey0, pkey1\n"
 #define STACK_SIZE (4 * 1024 * 1024)
 
 #ifdef LIBIA2_INSECURE
+#define pkey_mprotect insecure_pkey_mprotect
+static int insecure_pkey_mprotect(void *ptr, size_t len, int prot, int pkey) {
```
Comment (ayrtonm) [inline libia2/include/ia2.h:213]: Do we need this or is it just to simplify `INIT_RUNTIME_COMMON`?

**Comment by ayrtonm** [comment]
Just had a few nits and a question but otherwise LGTM.

File: libia2/include/ia2.h [hunk 2]
```diff
@@ -205,33 +209,38 @@ asm(".macro mov_mixed_pkru_eax pkey0, pkey1\n"
 #define STACK_SIZE (4 * 1024 * 1024)
 
 #ifdef LIBIA2_INSECURE
+#define pkey_mprotect insecure_pkey_mprotect
```
[the remark this answers is not in the document]
Comment (fw-immunant) [reply libia2/include/ia2.h:212]: Defining an alias (`int __attribute__((alias("insecure_pkey_mprotect"))) pkey_mprotect(void *ptr, size_t len, int prot, int key);`) results in multiple definitions of `pkey_mprotect`.

[the remark this answers is not in the document]
Comment (ayrtonm) [reply libia2/include/ia2.h:212]: shouldn't the names be backwards?

[the remark this answers is not in the document]
Comment (ayrtonm) [reply libia2/include/ia2.h:212]: not a big deal though

**Review by ayrtonm — APPROVED** [review]

**Pull request merged**